classes/payment/service_provider.php

```diff
@@ -97,7 +121,18 @@ public static function deliver_order(string $paymentarea, int $itemid, int $paym
 
         require_once($CFG->dirroot.'/enrol/wallet/lib.php');
         // Get the fake item in case of topping up the wallet.
-        $item = $DB->get_record('enrol_wallet_items', ['id' => $itemid], '*', MUST_EXIST);
+        $item = $DB->get_record('enrol_wallet_items', ['id' => $itemid], '*', IGNORE_MISSING);
+        if (!$item && $DB->get_manager()->table_exists('paygw_bank')) {
```

Comment:
Actually  we should not use missing item here as this is for payment or topup and the item should exist unless it is expired